app/javascript/themes/redesign/stylesheets/application.scss

```diff
@@ -1,3 +1,4 @@
+// === Vendor === //
```

Comment:
Can you explain your reasoning behind this change and the removal/changes of the other scss files?